src/ofdm.c

```diff
@@ -84,11 +84,11 @@ static const int8_t pilotvalues[] = {
    1, 1, 1, 1,-1, 1,-1, 1
 };
 
-/* static variables - used as storage for constants that are unchanged after init-time */
+/* constants that define modem waveform, unchanged after init-time,
+   but may vary for each instance of the modem */
 
-static struct OFDM_CONFIG ofdm_config;
+#define ofdm_tx_centre (ofdm->config.tx_centre)
 
```

Comment:
So the trick is to make each of the current statics a macro like this, so the code reads the same.  I've just changed one as a starting point (feel free to change them all if you like this approach).

Reply:
I'm wondering if it creates confusion though. Maybe just change all the variables to the new structure reference style? The only advantage would be keeping the old style.

I can do those changes pretty simple in the editor.

Reply:
Yep, or make them upper case ```#define OFDM_TX_CENTRE``` so we know they are macros.  The idea was to have something that was easy to read for the constants, rather that two deference levels.

However if you're comfy with just changing them all to the form ```ofdm->config.foo_bar``` I'm OK with that :+1:

Reply:
I'll see if I can get it done this evening.